python/simulation.py

```diff
@@ -859,6 +864,155 @@ def amplitude(self, point, component):
         return mp.eigenmode_amplitude(self.swigobj, swig_point, component)
 
 
+class Pade:
+    """
+    Pade is implemented as a class with a [`__call__`](#Pade.__call__) method,
+    which allows it to be used as a step function that collects field data from a given
+    point and runs [Pade](https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.pade.html)
+    on that data to extract an analytic rational function which models the frequency response.
```

Comment:
change "...which models the frequency response." to "...which approximates the frequency response."

Also, it would be useful to cite a reference on this topic such as the [wikipedia article](https://en.wikipedia.org/wiki/Pad%C3%A9_approximant) or a paper.

Reply:
https://github.com/NanoComp/meep/pull/2440/commits/6dd0939cac5f12cd11ece3b659ddd524f121f41e